client/scss/components/_modals.scss

```diff
@@ -63,6 +63,14 @@ $zindex-modal-background: 500;
   display: inline-block;
   vertical-align: middle;
   overflow: hidden;
+
+  button.close {
+    color: $color-white;
+
+    &:hover {
+      color: $color-white;
+    }
+  }
```

Comment:
I initially added new styles there but it wasn't specific enough and kept being overriden

Reply:
yeah this was because of the import order in `client/scss/core.scss`
the buttons were imported after modal, swapping these imports fixed it.

When trying override other css, if you have to fix with a more specific selector - that is usually an indicator that something else is wrong. Not always, and sometimes with a code-base this large there is no option. But always good to look at why you have to do it.